ui/pages/swaps/build-quote/build-quote.js

```diff
@@ -370,7 +373,7 @@ export default function BuildQuote({
 
   const swapYourTokenBalance = t('swapYourTokenBalance', [
     fromTokenString || '0',
-    fromTokenSymbol,
+    fromTokenSymbol || SWAPS_CHAINID_DEFAULT_TOKEN_MAP[chainId].symbol,
```

Comment:
As a more robust guard, could this be `fromTokenSymbol || SWAPS_CHAINID_DEFAULT_TOKEN_MAP[chainId]?.symbol || ''`? Just in case the chainId look up ever fails